pgtap/ksp/ksp/no_crash_test.pg

```diff
@@ -45,8 +51,8 @@ subs TEXT[];
 BEGIN
     -- one to one
     params = ARRAY[
-    '$$SELECT id, source, target, cost, reverse_cost  FROM edge_table$$'
-    ,'1::BIGINT',
+    '$$edges$$',
```

Comment:
Use the original:
```
$$SELECT id, source, target, cost, reverse_cost FROM edge_table$$'
```
because `edges` is a prepared statement of the query above.

Reply:
okay got it.